lib/paper_trail/record_trail.rb

```diff
@@ -67,14 +67,25 @@ def previous_version
 
     def record_create
       return unless enabled?
-      event = Events::Create.new(@record, true)
+
+      build_version_on_create(in_after_callback: true).tap do |version|
+        version.save!
+        versions.reset
+      end
+    end
+
+    # Returns the ready-for-save Version object built for a new record.
+    # Can be useful for manual Version saving, for instance as a part of
+    # bulk processing (https://github.com/zdennis/activerecord-import).
```

Comment:
I don't want to make any changes to our public API as part of an optimization PR. Please either:

1. inline this method, or
2. delete this comment and mark as `@api private`

Whichever you prefer.

I'd be happy to consider changes to our public API as a separate PR.